ranger-plugin/integration/service/src/test/java/org/apache/hawq/ranger/integration/service/tests/common/ServiceTestBase.java

```diff
@@ -67,7 +67,8 @@
     private static final String RANGER_URL = String.format("http://%s:%s/service/public/v2/api", RANGER_HOST, RANGER_PORT);
     private static final String RANGER_POLICY_URL = RANGER_URL + "/policy";
 
-    private static final int POLICY_REFRESH_INTERVAL = 6000;
+    private static final String POLICY_WAIT_INTERVAL_PROP_NAME = "policy.wait.interval.ms";
```

Comment:
could you explain what is the difference between ranger.plugin.hawq.policy.pollIntervalMs and policy.wait.interval.ms?  And in which xml file to config policy.wait.interval.ms?